tests/vm-prep

```diff
@@ -64,6 +64,12 @@ prepare()
 		$VIRSH net-undefine $NETWORK_NAME
 	fi
 
+    # In case a network has leaked in from elsewhere
+    if ip address show dev $NETWORK_NAME >/dev/null 2>/dev/null; then
+        ip link set $NETWORK_NAME down || true
+        brctl delbr $NETWORK_NAME || true
+    fi
+
```

Comment:
The indentation is off.

As the already existing cockpit1 was most likely created by libvirt already, is it right to remove it manually in that way? Does libvirt react to that appropriately, i. e. also remove its own configuration of it? Or will it try to restore it? It looks like this might only remove the actual bridge, but not libvirt's configuration of it?